src/Ui/Button.elm

```diff
@@ -84,15 +88,16 @@ view { onClick, color, emphasis } =
 {-| Returns a custom view of a button.
 -}
 customView :
```

Comment:
isn't it better to accept a record, so the user knows where the custom styling will be applied? nameless parameters says very little

Reply:
It's the overall button that this should be an escape hatch for and not anything internal. Do you still think it makes sense to add a record over this? It felt a bit superfluos, e.g.:

Ui.Button.view { buttonStyle = ..., buttonAttributes = ... }